test/controller.test.ts

```diff
@@ -30,9 +75,50 @@ contract('Controller', ([owner1, operator1, random]) => {
 
   describe('Account operator', () => {
     it('should set operator', async () => {
-      await controller.setOperator(operator1, true, {from: owner1})
+      await controller.setOperator(accountOperator1, true, {from: accountOwner1})
 
-      assert.equal(await controller.isOperator(owner1, operator1), true, 'Operator address mismatch')
+      assert.equal(await controller.isOperator(accountOwner1, accountOperator1), true, 'Operator address mismatch')
+    })
```

Comment:
can you test that it is initially not the operator, and then is the operator? 

Also test that once it is the operator and the permissions are revoked, it is no longer the operator? 

Also test that only the account owner can set the operator? The operator cannot set themselves/ remove themselves as operator